### ISSUE TITLE
Problem in expandrank.py file

### ISSUE THREAD

**Issue by neha9407** [boudinfl/pke#124]
There are double comma in line number 97 .

**Comment by gost-sniper** [comment]
indeed
```

Traceback (most recent call last):

  File "/home/ubuntu/.local/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 3331, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-10-246c2342e9e2>", line 1, in <module>
    from pke import compute_document_frequency

  File "/usr/local/lib/python3.7/dist-packages/pke/__init__.py", line 10, in <module>
    import pke.unsupervised

  File "/usr/local/lib/python3.7/dist-packages/pke/unsupervised/__init__.py", line 11, in <module>
    from pke.unsupervised.graph_based.expandrank import ExpandRank

  File "/usr/local/lib/python3.7/dist-packages/pke/unsupervised/graph_based/expandrank.py", line 97
    doc.load_document(input=input_file, , language=self.language, normalization=self.normalization)
                                        ^
SyntaxError: invalid syntax
```

**Comment by Oulaolay** [comment]
Hi @neha9407 @gost-sniper 
Sorry for the inconvenience I just fixed this. Don't hesitate if you find other errors.
Have a good day

**Comment by ygorg** [comment]
Fixed by #125
